app/static-pages/en/index.mdx

```diff
@@ -18,9 +23,49 @@ export const contentId = "home";
   example2Headline="Use powerful customizations"
   example2Description="With the help of custom filters and data segmentation, even complex issues can be visualized."
 />
-<Contribute
-  headline="Would you like to visualize your own data?"
-  description="Find out how you can integrate your data into the LINDAS Linked Data Service."
-  buttonLabel="Learn how"
-  buttonUrl="https://lindas.admin.ch/?lang=en"
-/>
+<Section>
+  <Contribute
+    headline="Would you like to visualize your own data?"
+    description="Find out how you can integrate your data into the LINDAS Linked Data Service."
+    buttonLabel="Learn how"
+    buttonUrl="https://lindas.admin.ch/?lang=en"
+  />
+  <div style={{ width: "1px", backgroundColor: "#e5e5e5" }}></div>
```

Comment:
Nit: `<Divider />`, this should be tied to MUI theming system (also in other pages) 👍